src/components/alpha-warning/__stories__/alpha-warning.mdx

```diff
@@ -0,0 +1,10 @@
+import { Controls, Canvas, Meta } from '@storybook/blocks';
+import * as AlphaWarningStories from './alpha-warning.stories';
+
+<Meta of={AlphaWarningStories} />
+
+# AlphaWarning
+
```

Comment:
Maybe let's write here that it's a version of Tip component and give a link to it?